<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkStylesheets</class>
 <widget class="QDialog" name="LinkStylesheets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Link StyleSheets</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="StylesheetsView">
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="MoveUp">
         <property name="toolTip">
          <string>Move the selected stylesheets up in priority.

Stylesheets that are listed first take precedence over later stylesheets.</string>
         </property>
         <property name="text">
          <string>Up</string>
         </property>
         <property name="arrowType">
          <enum>Qt::UpArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="MoveDown">
         <property name="toolTip">
          <string>Move the selected stylesheets down in priority.</string>
         </property>
         <property name="text">
          <string>Down</string>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LinkStylesheets</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LinkStylesheets</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
